ps_emailalerts.php

```diff
@@ -122,15 +122,18 @@ public function install($delete_params = true)
             Configuration::updateGlobalValue('MA_PRODUCT_COVERAGE', 0);
 
             $sql = 'CREATE TABLE IF NOT EXISTS `'._DB_PREFIX_.MailAlert::$definition['table'].'`
-				(
-					`id_customer` int(10) unsigned NOT NULL,
-					`customer_email` varchar(128) NOT NULL,
-					`id_product` int(10) unsigned NOT NULL,
-					`id_product_attribute` int(10) unsigned NOT NULL,
-					`id_shop` int(10) unsigned NOT NULL,
-					`id_lang` int(10) unsigned NOT NULL,
-					PRIMARY KEY  (`id_customer`,`customer_email`,`id_product`,`id_product_attribute`,`id_shop`)
-				) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci';
+                (
+                    `id_emailalert` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+                    `id_customer` int(10) unsigned NOT NULL,
+                    `customer_email` varchar(128) NOT NULL,
+                    `id_product` int(10) unsigned NOT NULL,
+                    `id_product_attribute` int(10) unsigned NOT NULL,
+                    `id_shop` int(10) unsigned NOT NULL,
+                    `id_lang` int(10) unsigned NOT NULL,
+                    `date_add` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                    PRIMARY KEY  (`id_customer`,`customer_email`,`id_product`,`id_product_attribute`,`id_shop`),
```

Comment:
If you add the `id_emailalert` the primary key must be updated. Add uniq constraint instead

Reply:
Hi @PierreRambaud , you are right. Modified now https://github.com/PrestaShop/ps_emailalerts/pull/40/commits/dc8f48160436b65b5ecb672c4fd842dec3ba5f04 :-)